src/scripts/main.js

```diff
@@ -1,3 +1,220 @@
 'use strict';
 
-// write code here
+const head = document.querySelector('thead');
+const table = document.querySelector('tbody');
+const body = document.querySelector('body');
+let column;
+let ascending = true;
+
+head.addEventListener('click', (ev) => {
+  const item = ev.target.closest('th');
+  const itemIndex = item.cellIndex;
+
+  if (column === itemIndex && ascending) {
+    return reSortRows([...table.rows], itemIndex).forEach(tr => {
+      table.append(tr);
+    });
+  }
+
+  sortRows([...table.rows], itemIndex).forEach(tr => {
+    table.append(tr);
+  });
+  column = itemIndex;
+});
+
+function num(str) {
+  return +str.replace(/[,$]/g, '');
+}
+
+function sortRows(arr, index) {
+  ascending = true;
+
+  return arr.sort((a, b) => {
+    if (isNaN(num(a.cells[index].innerText))) {
+      return a.cells[index].innerText.localeCompare(b.cells[index].innerText);
```

Comment:
since you use it twice and it looks big you can move `a.cells[index].innerText`, and `b.cells[index].innerText` to variables